### PR TITLE
Clarify callout options and update to markdown from Liquid

### PR DESCRIPTION
**Pull request by emangulo**
### Why:
In order to better align to the style guide and improve consistency in syntax.

### What's being changed (if available, include any code snippets, screenshots, or gifs):

- Updated the **Basic writing and formatting syntax** file to have the callouts be in markdown syntax instead of Liquid syntax.
- Updated the "Alerts" section of the same file to "Callouts" to align on the naming convention used in the style guide.
- Updated the **Style guide** file to include Important as a callout option as described in the **Basic writing and formatting syntax** file

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline (this link will be available after opening the PR).

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).

### CODE REVIEW

**Comment by nguyenalex836** [comment]
@emangulo Thanks so much for opening a PR! I'll get this triaged for review ✨

**Comment by felicitymay** [comment]
@emangulo - many thanks for flagging this discrepancy between the description of how to use Notes etc. in GitHub generally, and on the style guide for contributors to the user documentation for GitHub. I think you're right that we should use the same term for both.

I'm going to check in with our content strategy team on whether we should use the term "alerts" or "callouts" for both before I review this PR.

File: content/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax.md [hunk 1]
```diff
@@ -328,23 +305,18 @@ The footnote will render like this:
...
+> [!NOTE]
+> The position of a footnote in your Markdown does not influence where the footnote will be rendered. You can write a footnote right after your reference to the footnote, and the footnote will still render at the bottom of the Markdown. Footnotes are not supported in wikis.
 
 {% ifversion markdown-alerts %}
 
-## Alerts
+## Callouts
 
-Alerts are a Markdown extension based on the blockquote syntax that you can use to emphasize critical information. On {% data variables.product.product_name %}, they are displayed with distinctive colors and icons to indicate the significance of the content.
+Callouts are a Markdown extension based on the blockquote syntax that you can use to emphasize critical information. On {% data variables.product.product_name %}, they are displayed with distinctive colors and icons to indicate the significance of the content.
```
Comment (ethanpalm) [inline content/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax.md:315]:
```suggestion
Alerts are a Markdown extension based on the blockquote syntax that you can use to emphasize critical information. On {% data variables.product.product_name %}, they are displayed with distinctive colors and icons to indicate the significance of the content.
```

File: content/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax.md [hunk 2]
```diff
@@ -328,23 +305,18 @@ The footnote will render like this:
...
-Footnotes are not supported in wikis.
-
-{% endnote %}
+> [!NOTE]
+> The position of a footnote in your Markdown does not influence where the footnote will be rendered. You can write a footnote right after your reference to the footnote, and the footnote will still render at the bottom of the Markdown. Footnotes are not supported in wikis.
 
 {% ifversion markdown-alerts %}
 
-## Alerts
+## Callouts
```
Comment (ethanpalm) [inline content/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax.md:313]:
```suggestion
## Alerts
```

File: content/contributing/style-guide-and-content-model/style-guide.md [hunk 3]
```diff
@@ -47,16 +44,7 @@ Callouts should be concise. If the information consists of more than a couple of
 
 ### Callout types
```
Comment (ethanpalm) [inline content/contributing/style-guide-and-content-model/style-guide.md:45]:
```suggestion
### Alert types
```

File: content/contributing/style-guide-and-content-model/style-guide.md [hunk 4]
```diff
@@ -47,16 +44,7 @@ Callouts should be concise. If the information consists of more than a couple of
...
-
-#### Tip
-
-Recommendations, best practices or product hints. Tips contain non-essential information that users can follow at their discretion. Particularly useful in articles aimed at new users.
-
-For example, "[AUTOTITLE](/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile)" uses a tip callout to help users understand what to expect when they @mention an organization.
-
-> [!TIP]
-> When you @mention an organization, only those that you're a member of will autocomplete. You can still @mention organizations that you're not a member of, like a previous employer, but the organization name won't autocomplete for you.
+There are five types of callouts: Note, Tip, Important, Warning, and Caution.
```
Comment (ethanpalm) [inline content/contributing/style-guide-and-content-model/style-guide.md:47]:
```suggestion
We use four types of callouts: Note, Tip, Warning, and Caution.
```

File: content/contributing/style-guide-and-content-model/style-guide.md [hunk 5]
```diff
@@ -71,6 +59,20 @@ For example, "[AUTOTITLE](/code-security/secret-scanning/managing-alerts-from-se
...
+For example, "[AUTOTITLE](/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile)" uses a tip callout to help users understand what to expect when they @mention an organization.
+
+> [!TIP]
+> When you @mention an organization, only those that you're a member of will autocomplete. You can still @mention organizations that you're not a member of, like a previous employer, but the organization name won't autocomplete for you.
+
+#### Important
+
+> [!IMPORTANT]
+> Key information users need to know to achieve their goal.
+
```
Comment (ethanpalm) [inline content/contributing/style-guide-and-content-model/style-guide.md:75]:
```suggestion
```

File: content/contributing/style-guide-and-content-model/style-guide.md [hunk 6]
```diff
@@ -94,18 +96,25 @@ We use standard formatting and colors for different types of callouts across doc
...
 Tip:
 
 ```markdown
 > [!TIP]
 > Here's a suggestion.
 ```
 
-Note:
+Important:
```
Comment (ethanpalm) [inline content/contributing/style-guide-and-content-model/style-guide.md:114]:
```suggestion
```

File: content/contributing/style-guide-and-content-model/style-guide.md [hunk 7]
```diff
@@ -94,18 +96,25 @@ We use standard formatting and colors for different types of callouts across doc
...
 
 ```markdown
-> [!NOTE]
-> Keep this in mind.
+> [!IMPORTANT]
+> This is important.
 ```
```
Comment (ethanpalm) [inline content/contributing/style-guide-and-content-model/style-guide.md:118]:
```suggestion
```

File: content/contributing/style-guide-and-content-model/style-guide.md [hunk 8]
```diff
@@ -94,18 +96,25 @@ We use standard formatting and colors for different types of callouts across doc
...
 
 ```markdown
 > [!TIP]
 > Here's a suggestion.
 ```
 
-Note:
+Important:
 
 ```markdown
```
Comment (ethanpalm) [inline content/contributing/style-guide-and-content-model/style-guide.md:115]:
```suggestion
```

**Review by ethanpalm — COMMENTED** [review]
Hi @emangulo, thanks for creating this PR improve the consistency in syntax ✨ .

We should definitely use a single term, but since github.com uses "Alerts" and "Callouts" is just a GitHub Docs-specific term, we should bring things into alignment using "Alerts." I've added some suggestions to make those changes and I'll push a commit for the other paragraphs in the style guide that should be updated.

And while Important alerts are supported, we avoid using them in the Docs. I've suggested another change to make that more clear.

Thanks for highlighting the changes that needed to be made for these files and collaborating. Once the suggestions are incorporated, I can get this merged into the Docs 😄 .

**Comment by emangulo** [comment]
Hi @ethanpalm,
I have included your suggestions and also changed any additional references to callouts in the Alerts section of the style guide. Also removed the Important examples.

I have also updated this [Callout tags](https://docs.github.com/en/contributing/writing-for-github-docs/using-markdown-and-liquid-in-github-docs#callout-tags) section to _Alerts_

**Review by ethanpalm — DISMISSED** [review]
These changes look great, @emangulo. I'll get the branch up to date and make sure all checks are passing then merge the PR. Thank you for your contribution to the GitHub Docs!

File: content/contributing/writing-for-github-docs/using-markdown-and-liquid-in-github-docs.md [hunk 9]
```diff
@@ -48,29 +48,29 @@ This content is displayed on the {% data variables.product.prodname_docs %} site
...
 1. This is the next item.
 
-## Callout tags
+## Alerts
 
-Callouts highlight important information that users need to know. We use standard formatting and colors for four different types of callouts: notes, tips, warnings, and danger notices.
+Alerts highlight important information that users need to know. We use standard formatting and colors for four different types of Alerts: Note, Tip, Warning, and Caution.
 
-For information on when to use callouts, and how to format them in Markdown, see "[AUTOTITLE](/contributing/style-guide-and-content-model/style-guide#callouts)."
+For information on when to use alerts, and how to format them in Markdown, see "[AUTOTITLE](/contributing/style-guide-and-content-model/style-guide#callouts)."
```
Comment (ethanpalm) [inline content/contributing/writing-for-github-docs/using-markdown-and-liquid-in-github-docs.md:55]:
```suggestion
For information on when to use alerts, and how to format them in Markdown, see "[AUTOTITLE](/contributing/style-guide-and-content-model/style-guide#alerts)."
```

File: content/contributing/style-guide-and-content-model/style-guide.md [hunk 10]
```diff
@@ -122,9 +119,9 @@ Caution:
 > Be extremely careful.
 ```
 
-Liquid syntax for callouts is still supported and may still appear in older articles, but should not be used for new callouts.
+Liquid syntax for alerts is still supported and may still appear in older articles, but should not be used for new alerts.
 
-For more information on formatting callouts, see “Callouts” in "[AUTOTITLE](/contributing/syntax-and-versioning-for-github-docs/using-markdown-and-liquid-in-github-docs#callout-tags)."
+For more information on formatting alerts, see “Alerts” in "[AUTOTITLE](/contributing/syntax-and-versioning-for-github-docs/using-markdown-and-liquid-in-github-docs#callout-tags)."
```
Comment (ethanpalm) [inline content/contributing/style-guide-and-content-model/style-guide.md:124]:
```suggestion
For more information on formatting alerts, see “Alerts” in "[AUTOTITLE](/contributing/syntax-and-versioning-for-github-docs/using-markdown-and-liquid-in-github-docs#alerts)."
```

**Review by ethanpalm — APPROVED** [review]
There was an anchor in the footnotes section that was causing a failing test. Fixed and ready to merge now

**Pull request merged**